.github/workflows/typespec-validation-test.yaml

```diff
@@ -0,0 +1,27 @@
+name: TypeSpec Validation - Test
+
+on:
+  push:
+    branches:
+      - main
+      - typespec-next
+  pull_request:
+    branches:
+      - main
```

Comment:
I realize this is just a translation but is the any reason to scope to these branches as opposed to running on all branches?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/29167